### PR TITLE
add possibility to configure ETCD via url or proxy url 

### LINKED ISSUE #381: Configure ETCD connection via URL or Proxy

### PR DESCRIPTION
**Pull request by sulphur**
This adds a possibility to configure ETCD via URL or proxy as well as permits specifying SSL certificates for ETCD connection. 

Fixes #381

### CODE REVIEW

File: postgres-appliance/scripts/configure_spilo.py [hunk 1]
```diff
@@ -614,9 +614,20 @@ def get_dcs_config(config, placeholders):
...
+        config = {'etcd': {'proxy': placeholders['ETCD_PROXY']}}
     else:
         config = {}  # Configuration can also be specified using either SPILO_CONFIGURATION or PATRONI_CONFIGURATION
 
+    if 'CACERT' in placeholders :
+        config['etcd'].update({'cacert': placeholders['CACERT']})
+    if 'KEY' in placeholders :
+        config['etcd'].update({'key': placeholders['KEY']})
+    if 'CERT' in placeholders :
+        config['etcd'].update({'cert': placeholders['CERT']})
```
Comment (CyberDem0n) [inline postgres-appliance/scripts/configure_spilo.py]: Looks like you forgot about `ETCD_` prefix.

Comment (sulphur) [reply postgres-appliance/scripts/configure_spilo.py]: up

**Comment by CyberDem0n** [comment]
:+1:

**Comment by FxKu** [comment]
:+1:

**Pull request merged**